### PR TITLE
Recipient Model

### PR DESCRIPTION
**Pull request by guneetkb**

### CODE REVIEW

File: src/Recipients/models.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from phone_field import PhoneField
+
+# Create your models here.
+
+class Recipient(models.Model):
+     first_name = models.CharField(max_length=200)
```
Comment (askvortsov1) [inline src/Recipients/models.py:8]: Please remember that it's always preferable to represent these as users: that way, login can be added at a later time. Any class related to a user should be stored in the User model; attributes that relate to the user model, not a feature, should be handled by a custom user model. We shouldn't be replicating these across multiple models

File: src/Recipients/models.py [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+
+# Create your models here.
+
+class Recipient(models.Model):
+     first_name = models.CharField(max_length=200)
+     last_name = models.CharField(max_length=200)
+     email = models.EmailField(max_length=200, blank=True, null=True)
+     phone = PhoneField(blank=True, null=True)
+     age = models.PositiveIntegerField(null=True, blank=True)
+     parent_first_name = models.CharField(max_length=200)
```
Comment (askvortsov1) [inline src/Recipients/models.py:13]: Parents should be their own User, probably with a manytomany relationship (recipient -> user)

File: src/manage.py [hunk 3]
```diff
@@ -1,21 +1,21 @@
...
-
-def main():
-    os.environ.setdefault('DJANGO_SETTINGS_MODULE', 'cocoa_packs.settings')
-    try:
-        from django.core.management import execute_from_command_line
-    except ImportError as exc:
-        raise ImportError(
-            "Couldn't import Django. Are you sure it's installed and "
-            "available on your PYTHONPATH environment variable? Did you "
-            "forget to activate a virtual environment?"
```
Comment (askvortsov1) [inline src/manage.py:15]: What was changed here? Please revert this part.

File: src/Recipients/models.py (hunk 1 above)
Comment (guneetkb) [reply src/Recipients/models.py:8]: Alright! I'll look into that

File: src/Recipients/models.py (hunk 2 above)
Comment (guneetkb) [reply src/Recipients/models.py:13]: ok, I'll make the changes

File: src/manage.py (hunk 3 above)
Comment (guneetkb) [reply src/manage.py:15]: I don't think I changed anything here

Comment (askvortsov1) [reply src/manage.py:15]: Probably a line ending change. Please make sure you're using LF and not CRLF everywhere. And revert the change to this file so we don't confuse history

File: src/manage.py [hunk 4]
```diff
@@ -1,21 +0,0 @@
-#!/usr/bin/python3
-"""Django's command-line utility for administrative tasks."""
```
Comment (askvortsov1) [inline src/manage.py:2]: We don't want to delete the file, just revert the changes to it. Take a look at https://stackoverflow.com/questions/39459467/remove-a-modified-file-from-pull-request for instructions
